Packs/FeedZoom/Integrations/FeedZoom/FeedZoom.py

```diff
@@ -31,7 +31,8 @@ def build_iterator(self) -> List:
             A list of objects, containing the indicators.
         """
         result = []
-        r = self._http_request("GET", url_suffix="", full_url=self._base_url, resp_type="text")
+        session = requests.Session()
+        r = session.request("GET", self._base_url, headers={'User-Agent': 'Mozilla/5.0'}, verify=self._verify).text
```

Comment:
Do we have an instance? How was this fix tested?
It seems like the feed shouldn't have worked at all before the fix.